src/library/scala/StringContext.scala

```diff
@@ -154,6 +154,37 @@ case class StringContext(parts: String*) {
    */
   def raw(args: Any*): String = standardInterpolator(identity, args, parts)
 
+  /** Shorthand symbol constructor.
+   *
+   *  Here's an example of usage:
+   *  {{{
+   *    val abc = List(sym"a", sym"b", sym"c")
+   *    println(syms) // List('a, 'b, 'c)
+   *  }}}
+   *  This string-based shorthand, replaces the old built-in quote syntax:
+   *  {{{
+   *    val abc = List('a, 'b, 'c) // Quoted symbol literals dropped in 2.14
```

Comment:
I'll probably change the comment to read "Symbol literals were deprecated in 2.13"

Reply:
"dropped like a hot potato"

Reply:
"dropped like a bad habit"